### PR TITLE
docs: add page on created_at / updated_at timestamps (close #2880)

### LINKED ISSUE #2880: Make information about created_at and updated_at columns more easily accessible.
When looking around for information, or anything in the official docs (particularly when using the search), it's next to impossible to find out how to actually set up the created_at and updated_at columns.

It's not clear at all that these need to be set up explicitly using the "Frequently Used Columns" functionality.

Suggestion: Make sure that if someone searches the terms "createdAt", "created_at", "updatedAt", "updated_at", "lastUpdated", "last_updated" using the search in docs - useful information is actually shown.

Information about this functionality can pretty much only be found here: https://github.com/hasura/graphql-engine/issues/1077

image

### PR DESCRIPTION
**Pull request by marionschleifer**
### Description

Add a page on default timestamp values to make this information easier to find for users.

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Docs

### Related Issues

https://github.com/hasura/graphql-engine/issues/2880

### Added page

https://deploy-preview-5223--hasura-docs.netlify.app/graphql/manual/schema/default-values/created-updated-timestamps.html

### CODE REVIEW

**Comment by sezgi** [comment]
@marionschleifer should we also include instructions for `down.sql`?

File: docs/graphql/manual/schema/default-values/created-updated-timestamps.rst [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+    Click the ``Add column`` button.
+
+  .. tab:: CLI
+
+    :ref:`Create a migration manually <manual_migrations>` and add the following SQL statement to add a ``created_at`` timestamp field to the ``article`` table:
+
+    .. code-block:: plpgsql
+
+      ALTER TABLE ONLY "public"."article" ADD COLUMN "created_at" TIMESTAMP DEFAULT NOW();
```
Comment (sezgi) [inline docs/graphql/manual/schema/default-values/created-updated-timestamps.rst:43]: Let's clarify that this goes in `up.sql` and include the content for `down.sql` as well.

File: docs/graphql/manual/schema/default-values/created-updated-timestamps.rst [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+      RETURN NEW;
+      END;
+      $$ LANGUAGE plpgsql;
+
+      CREATE TRIGGER set_timestamp
+      BEFORE
+      UPDATE ON article
+      FOR EACH ROW
+      EXECUTE PROCEDURE trigger_set_timestamp();
+
```
Comment (sezgi) [inline docs/graphql/manual/schema/default-values/created-updated-timestamps.rst:110]: Let's clarify that this goes in `up.sql` and include the content for `down.sql` as well.

**Comment by marionschleifer** [comment]
@sezgi please see the changes.

**Pull request merged**